tools/warming_levels_approach.ipynb

```diff
@@ -0,0 +1,952 @@
+    "\n",
+    "**For this example, we will select 30-year slices centered around 2040, or the year when SSP3-7.0 reaches 2˚C** above pre-industrial levels, according to the IPCC weighted census projection.\n",
+    "\n",
+    "We can begin by going back to our `warming_levels()` object from the beginning, and accessing the original catalog data, via `wl.catalog_data`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bac55d5b-e6ed-4d4e-b4c2-64d56dcdc669",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Accessing our catalog data and selecting on the SSP3-7.0 scenario\n",
+    "time_horizon = wl.catalog_data.sel(scenario=\"Historical + SSP 3-7.0 -- Business as Usual\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33578dc4-b533-4ec8-aac9-f99a6e38f663",
+   "metadata": {},
+   "source": [
+    "We'll need to extract the gridcell of interest once again:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "131bb93a-d6e5-4ff7-a1c9-9424ba4dd50f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time_horizon = time_horizon.sel(lat=my_lat,lon=my_lon,method='nearest')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96d85ccf-a8ec-43b5-aa6e-69e7f3e738f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time_horizon = ck.load(time_horizon) # loading to disk for quicker RAM access, note that with a cluster, this will take ~5 min."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e9d6251-4ff7-4c08-b563-cfe37dd49331",
+   "metadata": {},
+   "source": [
+    "And then, we will extract the time slice for our time of interest."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3ea0814-2480-4119-aadd-33a18353d652",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Use the original time window from the beginning\n",
+    "ssp_data = time_horizon.sel(time=slice(str(2040-wl.wl_params.window),str(2040+(wl.wl_params.window-1))))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e36e78be-1c58-4327-8224-97634382eb66",
+   "metadata": {},
+   "source": [
+    "Now, we will use a similar process to generate the data needed for the new Global Warming Level framework as well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7fd326c-bb6a-4da9-a436-8aa5c57c1cf5",
+   "metadata": {},
+   "source": [
+    "## Step 2: Extract warming level data using new GWL method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca9653fb-e680-42f6-8bc6-9760405ac977",
+   "metadata": {},
+   "source": [
+    "Let's extract warming level data using the new GWL method. **We will once again select 30-year slices centered around 2040, or the year when SSP3-7.0 reaches 2˚C** above pre-industrial levels, according to the IPCC weighted census projection."
```

Comment:
I am confused why the 30-year slices are centered around 2040. Don't different models have different center years?

Reply:
Yes, this is true. I don't know why I wrote this. Re-writing now.